core/src/test/java/com/ibm/wala/core/tests/cha/AnalysisScopeTest.java

```diff
@@ -55,4 +70,79 @@ public void testBaseScope() throws IOException, ClassHierarchyException {
                 ClassLoaderReference.Application, "Ljava/awt/AlphaComposite")),
         "found unexpected class");
   }
+
+  @Test
+  public void testToJson() throws IOException {
+    AnalysisScope scope =
+        AnalysisScopeReader.instance.readJavaScope(
+            TestConstants.WALA_TESTDATA,
+            new FileProvider().getFile("GUIExclusions.txt"),
+            AnalysisScopeTest.class.getClassLoader());
+    Gson gson = new Gson();
+    Type type = new TypeToken<LinkedHashMap<String, Object>>() {}.getType();
+    LinkedHashMap<String, Object> map = gson.fromJson(scope.toJson(), type);
+    assertEquals(
+        List.of("java\\/awt\\/.*", "javax\\/swing\\/.*", "sun\\/awt\\/.*", "sun\\/swing\\/.*"),
+        map.get("Exclusions"));
+
+    @SuppressWarnings("unchecked")
+    Map<String, List<String>> loaders = (Map<String, List<String>>) map.get("Loaders");
+    Set<String> loaderKeys =
+        new HashSet<>(List.of("Primordial", "Extension", "Application", "Synthetic"));
+    assertEquals(loaders.keySet(), loaderKeys);
+    assertEquals(2, loaders.get("Primordial").size());
+    assertThat(loaders.get("Primordial"), hasItem("Nested Jar File:primordial.jar.model"));
+    assertEquals(1, loaders.get("Application").size());
+    assertThat(
+        loaders.get("Application").get(0), containsString("com.ibm.wala.core.testdata_1.0.0.jar"));
+    assertEquals(0, loaders.get("Extension").size());
+    assertEquals(0, loaders.get("Synthetic").size());
+  }
+
+  @Test
+  public void testToJsonCustom() throws IOException {
+    AnalysisScope scope;
+    scope = AnalysisScope.createJavaAnalysisScope();
+    AnalysisScope tempScope =
+        AnalysisScopeReader.instance.readJavaScope(
+            TestConstants.WALA_TESTDATA,
+            new FileProvider().getFile("GUIExclusions.txt"),
+            AnalysisScopeTest.class.getClassLoader());
```

Comment:
We don't need to create this temp scope just to test the exclusions.  Just skip testing the exclusions here